src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,73 @@
+.icon {
+  position: relative;
+  display: block;
+  width: $icon-width-small;
+  height: $icon-width-small;
+
+  @include onDesktop {
+    width: $icon-width-big;
+    height: $icon-width-big;
+  }
+
+  &--menu {
+    background-image: url(../images/menu.svg);
+
+    &:hover {
+      background-image: url(../images/menu-hover.svg);
+    }
+
+    @include onTablet {
+      height: $icon-height-small-option;
+    }
+  }
+
+  @include default-image-settings;
+  text-decoration: none;
+
+  &__tooltip {
+    position: absolute;
+    right: 100%;
+    top: 50%;
+    transform: translateY(-50%);
+    padding-right: 20px;
+
```

Comment:
check all scss files and remove spaces

   ```
    transform: translateY(-50%);
    padding-right: 20px;
    display: block;
    width: max-content;
    color: $color-white;
    text-shadow: $color-shadow-black-transparent;
    pointer-events: none;
    opacity: 0;
    transition: opacity 0.3s;
```
add spaces only between blocks, example:

```
&:hover &__tooltip {
    opacity: 1;
    pointer-events: all;
  }

  &__link {
```